Foundation/NSJSONSerialization.swift

```diff
@@ -23,6 +23,7 @@ extension JSONSerialization {
         public static let mutableContainers = ReadingOptions(rawValue: 1 << 0)
         public static let mutableLeaves = ReadingOptions(rawValue: 1 << 1)
         public static let allowFragments = ReadingOptions(rawValue: 1 << 2)
+        internal static let useReferenceNumericTypes = ReadingOptions(rawValue: 1 << 3)
```

Comment:
it might be a good idea to use some other value than `1 << 3` since it would be quite likely that we could add a public value that uses that definition.

my suggestion is use a higher value...